projects/1.19/assets/enders-delight/endersdelight/lang/zh_cn.json

```
@@ -0,0 +1,34 @@
+{
+  "block.endersdelight.chorus_crate": "箱装紫颂果",
+  "block.endersdelight.chorus_pie": "紫颂果派",
+  "block.endersdelight.endstone_stove": "末地石炉",
+  "block.endersdelight.stuffed_shulker_block": "填馅潜影贝",
+  "death.attack.wasWatered": "%1$s被浇了水",
+  "effect.endersdelight.phasing": "末影折跃",
+  "endersdelight.block.stuffed_shulker.use_shulker_bowl": "你需要一个碗才能吃这个",
+  "item.endersdelight.chorus_juice": "紫颂果汁",
+  "item.endersdelight.chorus_pie_slice": "紫颂果派切片",
+  "item.endersdelight.chorus_stew": "紫颂炖菜",
+  "item.endersdelight.chorus_stew_wood": "紫颂炖菜",
+  "item.endersdelight.crawling_sandwich": "蠕动三明治",
+  "item.endersdelight.crispy_skewer": "酥脆串",
+  "item.endersdelight.ender_paella": "末影海鲜饭",
+  "item.endersdelight.ender_paella_wood": "末影海鲜饭",
+  "item.endersdelight.ender_shard": "末影碎片",
+  "item.endersdelight.enderman_sight": "末影人眼球",
+  "item.endersdelight.endermite_stew": "末影炖肉",
+  "item.endersdelight.endermite_stew_wood": "末影炖肉",
```

```suggestion
  "item.endersdelight.endermite_stew": "末影螨煲",
  "item.endersdelight.endermite_stew_wood": "末影螨煲",
```
```suggestion
  "item.endersdelight.endermite_stew": "末影螨炖",
  "item.endersdelight.endermite_stew_wood": "末影螨炖",
```
上方为原版译法，下方为农夫乐事译法，推荐原版
endermite为原版术语